src/libraries/externals.csproj

```diff
@@ -26,7 +26,8 @@
 
   <!-- Setup the testing shared framework host -->
   <Target Name="SetupTestingHost"
-          AfterTargets="AfterResolveReferences">
+          AfterTargets="AfterResolveReferences"
+          Condition="'$(TestNativeAot)' != 'true'">
```

Comment:
This and the below targets probably wouldn't harm, but also it's possible to build a usable NativeAOT subset without building the host or corerun and having these fail to copy the useless files in that state is an annoyance.